config/config/common-config.yaml

```diff
@@ -352,6 +352,97 @@ prometheus:
         - name: elastisys
           pattern: ".*-elastisys-.*"
           limit: 80
+  diskAlerts:
```

Comment:
If one does not override any of these values, is the alerting behavior the same as before?

Reply:
Yes, they are the same.